### PR TITLE
Chunk and world-related mappings

### PR DESCRIPTION
**Pull request by IotaBread**
Includes some simulation distance mappings

### CODE REVIEW

File: mappings/net/minecraft/world/StructureHolder.mapping [hunk 1]
```diff
@@ -1 +1,14 @@
 CLASS net/minecraft/unmapped/C_xwhceuql net/minecraft/world/StructureHolder
+	METHOD m_bvnwwhpm getReferencesForStructure (Lnet/minecraft/unmapped/C_fpbotmvv;)Lit/unimi/dsi/fastutil/longs/LongSet;
+		ARG 1 structure
+	METHOD m_klfnsqrr setStructureReferences (Ljava/util/Map;)V
+		ARG 1 structureReferences
+	METHOD m_lwbwealm getStructureStart (Lnet/minecraft/unmapped/C_fpbotmvv;)Lnet/minecraft/unmapped/C_syfvaife;
```
Comment (AlsoSylv) [inline mappings/net/minecraft/world/StructureHolder.mapping:6]: Shouldn't this be StructureReferenceStart instead?

File: mappings/net/minecraft/world/StructureHolder.mapping [hunk 2]
```diff
@@ -1 +1,14 @@
...
+		ARG 1 structure
+	METHOD m_mfvlokwi setStructureStart (Lnet/minecraft/unmapped/C_fpbotmvv;Lnet/minecraft/unmapped/C_syfvaife;)V
```
Comment (AlsoSylv) [inline mappings/net/minecraft/world/StructureHolder.mapping:8]: Same here

File: mappings/net/minecraft/server/PlayerManager.mapping [hunk 3]
```diff
@@ -22,6 +23,9 @@ CLASS net/minecraft/unmapped/C_wxtgmgid net/minecraft/server/PlayerManager
 	FIELD f_wfmkgkxe cheatsAllowed Z
 	FIELD f_wsxdzigk DATE_FORMATTER Ljava/text/SimpleDateFormat;
 	FIELD f_xwqjfjqr bannedIps Lnet/minecraft/unmapped/C_jkzjkajz;
+	FIELD f_zujxwecs ALLOW_LOGOUT_TO_ACT_LIKE_ELEVATOR Z
```
Comment (AlsoSylv) [inline mappings/net/minecraft/server/PlayerManager.mapping]: ALLOW_LOGOUT_ELEVATOR would probably be better than ALLOW_LOGOUT_TO_ACT_LIKE_ELEVATOR

File: mappings/net/minecraft/world/StructureHolder.mapping (hunk 1 above)
Comment (IotaBread) [reply mappings/net/minecraft/world/StructureHolder.mapping:6]: `Chunk` is the only implementation of `StructureHolder`, so I named the methods according to the corresponding field names (See [`Chunk#structureStarts`](https://github.com/QuiltMC/quilt-mappings/pull/25/commits/ac27cbf4be3ff85f56fdb27af27ae37d3ed03a17#diff-10779271a84fc7c569faebba54f00f8f37eb30e4e3f38d03568be616597df298R6) and [`Chunk#structureReferences`](https://github.com/QuiltMC/quilt-mappings/pull/25/commits/ac27cbf4be3ff85f56fdb27af27ae37d3ed03a17#diff-10779271a84fc7c569faebba54f00f8f37eb30e4e3f38d03568be616597df298R18), both have the same name in Mojmap and Yarn IIRC)

Comment (AlsoSylv) [reply mappings/net/minecraft/world/StructureHolder.mapping:6]: Both do mean about the same in yarn and MM, looks like these are proper names
(MM is `getStartForFeature` for reference)

**Review by AlsoSylv — APPROVED** [review]
Looks good to me, didn't find any issues

File: mappings/net/minecraft/world/chunk/ChunkSection.mapping [hunk 4]
```diff
@@ -17,6 +31,11 @@ CLASS net/minecraft/unmapped/C_fgfsorua net/minecraft/world/chunk/ChunkSection
 	METHOD m_lmfwzupo hasRandomTicks ()Z
 	METHOD m_olwomvfb hasAny (Ljava/util/function/Predicate;)Z
 		ARG 1 predicate
+	METHOD m_oxqqkhup (Lnet/minecraft/unmapped/C_aywougja;Lnet/minecraft/unmapped/C_ygdawqho$C_veuekhcq;II)V
+		ARG 1 source
+		ARG 2 sampler
+		ARG 3 x
+		ARG 4 z
```
Comment (OroArmor) [inline mappings/net/minecraft/world/chunk/ChunkSection.mapping:38]: I cant find this method in linkie or cozy, will need to investigate here

**Review by NoComment1105 — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Pull request merged**